### PR TITLE
Started Documentation of Introduction and code structure

### PR DESCRIPTION
**Pull request by jimrojerTNO**
This PR is in a really rough state, this one is created so that we have a place to comment and review things.

For now I mainly focussed on the structure of the rationale. Let's see if we can align on that and then decide if we improve this text or write (partially) from scratch.

### CODE REVIEW

File: docs/theory/introduction.rst [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+
+What is MESIDO?
+---------------
+
+MESIDO is a python package that allows users to define techno-economic optimization workflows (or use the available ones). Users can define themselves which KPIs they wish to optimize for, e.g. cost or business case optimization.
+Furthermore, the user can select a physics fidelity level he/she needs for the analyses. Where the physics are typically approximations of steady-state conditions, the modelling is done such that the solutions are close and conservative.
+The inclusion of the physics approximations allows the user to make more design choices compared to the tools used in initial feasibility.
+However, this comes at the cost of the user needing to provide more information of the energy system in order to use MESIDO in an effective way, hence MESIDO is proposed as a sequential tool for analysis.
+
+Under the hood MESIDO is a Mixed Integer Linear Problem (MILP) formulation. This means that the physics and financial models are limited to linear constraints.
```
Comment (FJanssen-TNO) [inline docs/theory/introduction.rst:30]:
```suggestion
Under the hood, MESIDO is a Mixed Integer Linear Problem (MILP) formulation. This means that the physics and financial models are limited to linear constraints.
```

File: docs/theory/introduction.rst [hunk 2]
```diff
@@ -1,23 +1,39 @@
...
+The computational power advantages of MILP optimizations allows MESIDO to optimize Multi Energy/Multi Commodity networks in an integral manner.
+Nevertheless, the modelling limitations that MILP have will in the fast majority of cases require optimized networks to be detailed out with simulation tools before closing business cases and going to exploitation.
```
Comment (FJanssen-TNO) [inline docs/theory/introduction.rst:32]:
```suggestion
Nevertheless, the modelling limitations of MILP in the fast majority of cases, require optimized networks to be detailed out with simulation tools before closing business cases and going to exploitation.
```

**Review by FJanssen-TNO — COMMENTED** [review]
I agree with the overall structure of the explanation only some small rephrasing suggestions

**Comment by jimrojerTNO** [comment]
@FJanssen-TNO @KobusVanRooyen can one of you approve and then press merge?

**Review by FJanssen-TNO — APPROVED** [review]

**Pull request merged**